commands/FBClassDump.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+
+import os
+import re
+import string
+
+import lldb
+import fblldbbase as fb
+import fblldbobjcruntimehelpers as runtimeHelpers
+
+def lldbcommands():
+  return [
+    FBPrintMethods()
+  ]
+
+class FBPrintMethods(fb.FBCommand):
+  def name(self):
+    return 'pmethods'
+
+  def description(self):
+    return 'Print the class instance methods.'
+
+  def options(self):
+    return [
+      fb.FBCommandArgument(short='-a', long='--all', arg='all', help='If display all methos include class and instance methods', default=False, boolean=True),
```

Comment:
I find the combination of '-a' and '-c' to be a bit counter-intuitive. What about '-i/--instance' for instance and '-c/--class' for class and if neither is specified then it defaults to both. This ensures that there aren't any combinations that don't make sense.